matrix.yaml

```diff
@@ -29,6 +30,14 @@ exclude:
   # Exclusions from CUDA's OS support matrix
   - LINUX_VER: "ubuntu22.04"
     CUDA_VER: "11.4.3"
+  - LINUX_VER: "ubuntu24.04"
+    CUDA_VER: "11.4.3"
+  - LINUX_VER: "ubuntu24.04"
+    CUDA_VER: "11.8.0"
+  - LINUX_VER: "ubuntu24.04"
+    CUDA_VER: "12.0.1"
+  - LINUX_VER: "ubuntu24.04"
+    CUDA_VER: "12.2.2"
```

Comment:
I'm guessing we plan to take these exclusions out once testing is complete and they're confirmed to work?